tests/configm_test.sh

```diff
@@ -43,10 +43,10 @@ function setUp()
   rm -rf "$TMP_TEST_DIR"
 
   mkdir -p "$TMP_TEST_DIR"
-  cd "$TMP_TEST_DIR"
+  cd "$TMP_TEST_DIR" || exit
```

Comment:
In the case of tests, use `fail`. See https://github.com/kworkflow/kworkflow/pull/312/files for a reference

Reply:
@rodrigosiqueira thinking about it again, maybe we should fail and return. If we are `cd`ing into a temporary directory, it means we want to pollute that folder. If we fail to `cd`, we might end up polluting our main directory (or worse, deleting something from it). Exiting is bad since it cancels all other test functions in this file, but canceling one function seems reasonable. What do you think?

Reply:
@kwy95

Reply:
failing doesnt lead to automatically returning? @joaoseckler